src/lib/background/background.js

```diff
@@ -82,7 +102,6 @@ export class Background {
     const updateCount = this.pageStore.getPageList()
       .filter(isItemChanged).length;
 
-    browser.browserAction.setIcon({path: activeIcon});
```

Comment:
Make this function `refreshIcon(icon)`, and pass in either scanIcon or defaultIcon.